static/js/tools/map/chart.tsx

```diff
@@ -89,10 +89,11 @@ export function Chart(props: ChartProps): JSX.Element {
   const statVarInfo = props.statVar.value;
   const [errorMessage, setErrorMessage] = useState("");
   const [denomInput, setDenomInput] = useState(props.statVar.value.denom);
-  const title = getTitle(
+  const chart_title=   getTitle(
     Array.from(props.dates),
     statVarInfo.info.title ? statVarInfo.info.title : statVarInfo.dcid
   );
+  const title = props.statVar.value.perCapita? 'Per Capita ' + chart_title : chart_title; 
```

Comment:
please follow the convention from scatter (per capita comes after the sv name)

this edit also belongs to the getTitle function, passing in props.statVar.value.perCapita

also, as an fyi - we use camelcase in typescript

Reply:
Thank you for the review. I have moved 'Per Capita' to the end of the StatVar's name with the edit being made in the `getTitle` function.

Reply:
With the edit in the `getTitle` function, line 97 should be removed. And maybe change `chart_title` back to `title`?

Reply:
Thanks for the review, Jennifer. I have cleaned up the code. This particular issue when I tried resolving a git divergent branches error. :)